action/protocol/staking/handlers.go

```diff
@@ -775,6 +776,83 @@ func (p *Protocol) handleCandidateUpdate(ctx context.Context, act *action.Candid
 	return log, nil
 }
 
+func (p *Protocol) handleTypedCandidateRegister(ctx context.Context, act typedCandidateRegister, csm CandidateStateManager, esm *typedCandidateStateManager,
```

Comment:
where is `handleTypedCandidateRegister` called?

Reply:
Can similar code be shared with `handleCandidateRegister`

Reply:
`handleTypedCandidateRegister` will be called when introducing a specific register action

Reply:
`handleTypedCandidateRegister` uses different candidate state storage than `handleCandidateRegister`

Reply:
> `handleTypedCandidateRegister` will be called when introducing a specific register action

Add TODO?